### ISSUE TITLE
Unknown string format error parsing dates

### ISSUE THREAD

**Issue by sminnee** [singer-io/tap-github#20]
Importing github data into stitch gives the following error. At a glance it seems like there's an unrecognised date format.

```
> tap-github -c github-config.json -p github-properties.json | target-stitch -c stitch-config.json 
INFO Using Stitch import URL https://api.stitchdata.com/v2/import/batch
INFO Sending version information to stitchdata.com. To disable sending anonymous usage data, set the config parameter "disable_collection" to true
INFO METRIC: {"type": "timer", "metric": "http_request_duration", "value": 0.8320388793945312, "tags": {"endpoint": "assignees", "http_status_code": 200, "status": "succeeded"}}
CRITICAL ('Unknown string format:', '4Y-04-11T05:04:42.495197Z')
Traceback (most recent call last):
  File "/usr/local/bin/target-stitch", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/target_stitch/__init__.py", line 528, in main
    raise exc
  File "/usr/local/lib/python3.6/site-packages/target_stitch/__init__.py", line 514, in main
    main_impl()
  File "/usr/local/lib/python3.6/site-packages/target_stitch/__init__.py", line 507, in main_impl
    args.batch_delay_seconds).consume(reader)
  File "/usr/local/lib/python3.6/site-packages/target_stitch/__init__.py", line 413, in consume
    self.handle_line(line)
  File "/usr/local/lib/python3.6/site-packages/target_stitch/__init__.py", line 373, in handle_line
    message = singer.parse_message(line)
  File "/usr/local/lib/python3.6/site-packages/singer/messages.py", line 183, in parse_message
INFO METRIC: {"type": "counter", "metric": "record_count", "value": 24, "tags": {"endpoint": "assignees"}}
    time_extracted = dateutil.parser.parse(time_extracted)
  File "/usr/local/lib/python3.6/site-packages/dateutil/parser/_parser.py", line 1312, in parse
    return DEFAULTPARSER.parse(timestr, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/dateutil/parser/_parser.py", line 607, in parse
    raise ValueError("Unknown string format:", timestr)
ValueError: ('Unknown string format:', '4Y-04-11T05:04:42.495197Z')
INFO METRIC: {"type": "timer", "metric": "http_request_duration", "value": 0.5094437599182129, "tags": {"endpoint": "commits", "http_status_code": 200, "status": "succeeded"}}
INFO METRIC: {"type": "counter", "metric": "record_count", "value": 0, "tags": {"endpoint": "commits"}}
```

**Comment by sminnee** [comment]
Looks like this is the cause https://github.com/singer-io/singer-python/issues/59

**Comment by nick-mccoy** [comment]
We just merged [#69](https://github.com/singer-io/singer-python/pull/69) in singer-python that resolves this issue.  Our implementation is similar to what was done in tap-codat [#3](https://github.com/singer-io/tap-codat/pull/3/files) and works on all platforms.
